Engage Audience Destination Update [DOC-892] (#6520)
segmentio/segment-docs: src/engage/audiences/send-audience-data.md

---
title: Send Audience Data to Destinations
plan: engage-foundations
---

> With the help of Sources and Destinations in Segment's catalog, you can create and send audiences and computed traits to third-party services.
> 
> Segment's Connections pipeline first collects and sends events from your source to your destination. Built on top of Connections, Engage then uses the same source events to let you create audiences and computed traits within Segment. You can then send the audience or computed trait you've built to your destination(s).

Reviewer: ```suggestion
With the help of sources and destinations in Segment's catalog, you can create and send audiences and computed traits to third-party services.

Segment's Connections pipeline first collects and sends events from your source to your destination. Built on top of Connections, Engage then uses the same source events to let you create audiences and computed traits within Segment. You can then send the audience or computed trait you've built to your destination(s).
```

> info ""
> Because Engage only sends audiences and computed traits to destinations, it doesn't replace a standard event pipeline. Connect a source directly to a destination if you want the destination to receive all events that Segment gathers.

## Connect your audience to a destination

Once you've previewed your audience, you can choose to connect it to a destination or keep the audience in Segment and export it as a CSV file download.

> When you create an audience, Segment starts syncing your audience to the destinations you selected. Audiences are either sent to destinations as a boolean user-property or a user-list, depending on what the destination supports. Read more about [supported Destinations](/docs/engage/using-engage-data/#compatible-engage-destinations) in the Engage documentation.

Reviewer: ```suggestion
When you create an audience, Segment starts syncing your audience to the destinations you selected. Audiences are either sent to destinations as a boolean user-property or a user-list, depending on what the destination supports. Read more about [supported destinations](/docs/engage/using-engage-data/#compatible-engage-destinations) in the Engage documentation.
```

> For account-level audiences, you can send either a [Group](/docs/connections/spec/group) call and/or [Identify](/docs/connections/spec/identify) call. Group calls will send one event per account, whereas Identify calls will send an Identify call for each user in the account. This means that even if a user hasn't performed an event, Segment will still set the account-level computed trait on that user.

Reviewer: ```suggestion
For account-level audiences, you can send either a [Group](/docs/connections/spec/group) call and/or [Identify](/docs/connections/spec/identify) call. Group calls send one event per account, whereas Identify calls send an Identify call for each user in the account. This means that even if a user hasn't performed an event, Segment will still set the account-level computed trait on that user.
```

1. Navigate to **Engage > Audiences**, then select the audience you want to connect.
2. From the audience's overview page, click **+ Add destination**.
3. Select the destination you want to connect to, then click **Add destination**. 
4. Segment then begins its initial sync to the destination.

## View connected destinations

You can view a list of an audience's connected destinations in the destination list table of the audience overview tab. 

![The Engage Destinations table showing two connected destinations](/docs/engage/images/destinations_table.png)

The Destinations table contains information about the destination's matching mappings, status, and sync status.

### Matching mappings

[Actions destinations](/docs/connections/destinations/actions/) have mappings that can receive granular data from your audience. The **Matching mappings** column shows the number of mappings that match the data coming from the audience, as well as the number of enabled and disabled mappings. See [Working with mappings](#working-with-mappings) for more information.

The Matching mappings column will show `Not applicable` for classic destinations.

### Status columns

> The **Destination status** column will show either `Connected`, `Disconnected`, or `Disabled`: 

Reviewer: ```suggestion
The **Destination status** column shows `Connected`, `Disconnected`, or `Disabled`: 
```
